lib/http.dart

```diff
@@ -79,6 +79,27 @@ class HTTP {
         .map<Accommodation>((json) => Accommodation.fromJson(json))
         .toList();
   }
+
+  Future<Response> addNewPlace(Map<String, dynamic> map) {
+    return client.post(
+      'myplaces',
+      data: map,
+    );
+  }
```

Comment:
replace with :
  Future<Accommodation> addNewPlace(Map<String, dynamic> map) async {
    var response = await client.post(
      'myplaces',
      data: map,
    );
    return Accommodation.fromJson(response.data);
  }

Reply:
It's above addNewPlace :)